lib/git-fastclone.rb

```diff
@@ -441,5 +448,11 @@ def with_git_mirror(url)
     def usage
       'Usage: git fastclone [options] <git-url> [path]'
     end
+
+    private def trigger_pre_clone_hook(url, mirror, attempt_number)
+      return if Dir.exist?(mirror) || !options.include?(:pre_clone_hook)
+
+      popen2e_wrapper(options[:pre_clone_hook], url, mirror, attempt_number.to_s, quiet: !verbose)
```

Comment:
Do we need to stringify the url and mirror params as well? I see are doing that above on the clone operation. If we know they'll be strings, then this is better, but I wanted to check.

Reply:
Sure, I can do that but I checked that in runtime and url and mirror are strings already.